test/test_heuristic.cpp

```diff
@@ -156,8 +240,14 @@ TEST_P(HeuristicTest5Q, Static) {
   ibmqYorktownMapper->dumpResult(GetParam() + "_heuristic_qx4_static.qasm");
   ibmqYorktownMapper->printResult(std::cout);
   ibmqLondonMapper->map(settings);
+  auto& result = ibmqYorktownMapper->getResults();
+  EXPECT_LE(result.heuristicBenchmark.effectiveBranchingFactor,
+            result.heuristicBenchmark.averageBranchingFactor);
   ibmqLondonMapper->dumpResult(GetParam() + "_heuristic_london_static.qasm");
   ibmqLondonMapper->printResult(std::cout);
+  result = ibmqLondonMapper->getResults();
+  EXPECT_LE(result.heuristicBenchmark.effectiveBranchingFactor,
+            result.heuristicBenchmark.averageBranchingFactor);
   SUCCEED() << "Mapping successful";
```

Comment:
Looking at all the lines that needed to be changed here, we should really be thinking about refactoring these tests to become less redundant.
It must be possible to write slightly more complicated test fixtures that allow to unify these tests in some fashion.
(Probably nothing for this PR)

Reply:
You are probably right, but I am really no expert on Google Test. So maybe we can find someone else for that, who is more familiar with the framework.

Reply:
Yeah, let's do that. I'll put it on my list of todos.